main_small_batch.py

```diff
@@ -8,17 +15,31 @@
 from pyomo.gdp import Disjunct, Disjunction
 from pyomo.util.infeasible import log_infeasible_constraints
 
-from gdp.dsda.dsda_functions import (external_ref, generate_initialization,
-                                     get_external_information,
-                                     initialize_model,
-                                     solve_complete_external_enumeration,
-                                     solve_subproblem, solve_with_dsda,
-                                     solve_with_gdpopt, solve_with_minlp,
-                                     visualize_dsda)
+from gdp.dsda.dsda_functions import (
+    external_ref,
+    generate_initialization,
+    get_external_information,
+    initialize_model,
+    solve_complete_external_enumeration,
+    solve_subproblem,
+    solve_with_dsda,
+    solve_with_gdpopt,
+    solve_with_minlp,
+    visualize_dsda,
+)
 from gdp.small_batch.gdp_small_batch import build_small_batch
 
 
 def problem_logic_batch(m):
+    """
+    This function returns the logic expressions to be used in the disjunctive constraints of the D-SDA approach for the small batch problem.
+
+    Args:
+        m(pyomo.ConcreteModel): Pyomo model of the small batch problem
+
+    Return:
+        logic_expr(list): List of logic expressions to be used in the disjunctive constraints
```

Comment:
There is the same space missing between the name of the return (logix_expr) and the type ( (list) ).
Please add it and make sure that black is not removing it.

Reply:
The black formatting is not removing and I adjust it adding the space.

Reply:
The space is still missing!

Reply:
This is not resolved